doc/api/child_process.md

```diff
@@ -39,6 +39,12 @@ without blocking the Node.js event loop. The [`child_process.spawnSync()`][]
 function provides equivalent functionality in a synchronous manner that blocks
 the event loop until the spawned process either exits or is terminated.
 
+*Note*: On some operating systems, the [`child_process.spawn()`][] method
+performs memory operations synchronously before decoupling the event loop
```

Comment:
I'd be more specific and write "On UNIX-like operating systems" and tweak the wording to clarify that it's the underlying fork() system call that is responsible for copying memory, not `child_process.spawn()` (directly.)

Reply:
I like "UNIX-like", but I don't think we need to go into detail on the root cause. The V8 link explains things.